templates/portfolio/Tutorial/styles.css

```diff
@@ -0,0 +1,105 @@
+
+.portfolio-header {
+  position: relative;
+  background-color: #D8D8D8;
+  background-image: url(images/header-bg.jpg);
+}
+
+.mdl-layout__header-row {
+  padding: 0;
+  justify-content: center;
+}
+
+.portfolio-navigation-row {
+  background-color: rgba(0, 0, 0, 0.08);
+  text-transform: uppercase;
+  height: 45px;
+}
+
+.portfolio-navigation-row  .mdl-navigation {
+  text-align: center;
+  max-width: 900px;
+  width: 100%;
+}
+
+.portfolio-navigation-row .mdl-navigation__link {
+  flex: 1;
+  line-height: 42px;
+}
+
+.mdl-layout__header .mdl-layout__drawer-button {
```

Comment:
This should probably be `.portfolio-header .mdl-layout__drawer-button`, for the same reasons as above.